examples/knative/go/kn-go-tagging/tagging/tagging_test.go

```diff
@@ -0,0 +1,97 @@
+package tagging
```

Comment:
Can we get some unit tests around the handler please?

Reply:
I was able to get a test where authentication to vsphere failed but I can't figure out how to do tests for the REST (tagger manager) client. Any ideas?

Reply:
Very helpful, thank you! Tests are finally up.